src/pages/_app.jsx

```diff
@@ -111,6 +111,16 @@ const WrappedApp = ({ Component, pageProps }) => {
         );
       }
 
+      if (httpError === 424) {
+        return (
+          <NotFoundPage
+            title='Terms agreement required'
+            subTitle={'You don\'t have access to the platform.'}
+            hint='First please go to data management and accept the privacy policy.'
```

Comment:
remove the "please" from here -- see this discussion about uses of please in messages to the user: https://ux.stackexchange.com/questions/10853/should-an-interface-ever-say-please

Reply:
You can rename the hint message to say "To get access to the platform, go to data management and accept the privacy policy."